driver-core/src/main/com/mongodb/client/model/expressions/DocumentExpression.java

```diff
@@ -25,14 +25,14 @@
 import static com.mongodb.client.model.expressions.Expressions.ofMap;
 import static com.mongodb.client.model.expressions.MqlUnchecked.Unchecked.PRESENT;
 import static com.mongodb.client.model.expressions.MqlUnchecked.Unchecked.TYPE;
+import static com.mongodb.client.model.expressions.MqlUnchecked.Unchecked.TYPE_ARGUMENT;
 
 /**
  * Expresses a document value. A document is an ordered set of fields, where the
  * key is a string value, mapping to a value of any other expression type.
  */
```

Comment:
In all previous PRs this doc was told to be considered a draft and not reviewed. Here it should be reviewed, but there is no way for reviewers to know that they should review it. I wonder what other docs like this one we have.

Reply:
I checked, and it is this one and String. We documented all the classes, and only a few methods of the original classes (boolean, and some others, which should have been fixed).